typescript-getting-started/README.md

```diff
@@ -0,0 +1,143 @@
+## Cloud Functions for Firebase: getting started with TypeScript
+
+This is an example Firebase project for using TypeScript with
+[Cloud Functions for Firebase](https://firebase.google.com/products/functions)
+
+Pre-requistes:
+
+```
+npm install -g firebase-tools
+```
+
+Also, I highly recommend using [yarn](https://yarnpkg.com),
+which locks the versions of your npm modules.  I also like to use
+[homebrew](https://brew.sh/) to install dependencies on the Mac.  These aren't
+required for Cloud Functions or Firebase, but are used in the instructions.
+
+```
+brew install yarn
```

Comment:
`npm install -g yarn` is more likely to work on the developer's machine. Then the dev wouldn't have to go through the work of a heavy install to try out typescript either.

Reply:
I'll just skip the yarn install, there are details at the yarn link